Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -805,6 +802,7 @@ private bool CheckForCollisionBetween()
             // INCLUDING[!!!] THE LAST AND NEW POSITIONS!
             var cells = GenSight.PointsOnLineOfSight(lastPosIV3, newPosIV3).Union(new[] { lastPosIV3, newPosIV3 }).Distinct().OrderBy(x => (x.ToVector3Shifted() - LastPos).MagnitudeHorizontalSquared());
```

Comment:
This should come *after* we find our closest possible intersection point and use that as the end point for `PointsOnLineOfSight`

Reply:
Done